plugins/GSdx/GSRendererHW.cpp

```diff
@@ -121,8 +118,9 @@ void GSRendererHW::Reset()
 
 void GSRendererHW::VSync(int field)
 {
-	//Check if the frame buffer width or display width has changed
-	SetScaling();
+	//Set Scaling only for native scaling resolutions and only after the Display rectangle is stabilized.
+	if(GetDisplayRect().height() > 1 && m_upscale_multiplier)
```

Comment:
Maybe put those check in the SetScaling function? m_upscale_multiplier is already checked by the way.